SU2_CFD/src/solvers/CTurbSSTSolver.cpp

```diff
@@ -290,6 +293,15 @@ void CTurbSSTSolver::Source_Residual(CGeometry *geometry, CSolver **solver_conta
   SU2_OMP_FOR_DYN(omp_chunk_size)
   for (unsigned long iPoint = 0; iPoint < nPointDomain; iPoint++) {
 
+    /*--- If fixed values apply, there is no need to compute the source term. ---*/
+    if(config->GetTurb_Fixed_Values()){
+      const su2double* velocity_inf = config->GetVelocity_FreeStreamND();
+      su2double sp =
+        GeometryToolbox::DotProduct(nDim, geometry->nodes->GetCoord(iPoint), velocity_inf)
```

Comment:
Would it be appropriate to take the coordinate relative to the reference point provided in the config?
I guess in practice one would set the plane offset very large, but you never know what folks will do with your code (for better or worse).

Reply:
Hmmmm you now have to check for zero velocity in two places as well...

Reply:
Right ... I think I will revert the changes in `CTurbSSTSolver::Source_Residual`, so that the source residual is computed even if the residual is overwritten later.

Reply:
How is a reference point specified in the config?

Reply:
Probably the `REF_ORIGIN_MOMENT_X`, _Y, _Z. Yes giving the shift relative to these can be a sensible choice, let's discuss in the meeting.